src/host/ut_host/UnicodeLiteral.hpp

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-#define REMOTE_STRING L"�remote �npipe:pipe=foo,server=bar�\t"
+#define REMOTE_STRING L"–remote “npipe:pipe=foo,server=bar”\t"

```

Comment:
Holy frick. We should **absolutely** just make these boring-quotes.

```suggestion
#define REMOTE_STRING L"–remote \"npipe:pipe=foo,server=bar\"\t"
```

Reply:
I think this was on purpose but when I went to go find the test that used it, it looks like nothing uses it anymore. So I'm going to just delete it.